package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "jsfs-precourse-set-timeout-loop",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "index.ts",
```

Comment:
Nice catch!

Formally, there is no main in this app, so this property should have been deleted by me. I just kept most of the defaults set by `npm init -y`.